src/Components/fe-login/FeLogin.js

```diff
@@ -20,6 +21,11 @@ export class FeLogin extends LocalizeMixin(LitElement) {
     `;
   }
 
+  constructor() {
```

Comment:
constructor is not needed, bcz we are not doing anything from `isError`

Reply:
okay will remove the constructor